packages/react-resizable-panels/src/PanelResizeHandleRegistry.ts

```diff
@@ -73,6 +73,10 @@ export function registerResizeHandle(
     if (count === 1) {
       ownerDocumentCounts.delete(ownerDocument);
     }
+
+    const dummyEvent = { target: null, x: 0, y: 0 };
+    recalculateIntersectingHandles(dummyEvent);
+    updateCursor();
```

Comment:
Would you prefer I pass a dummy event to `recalculateIntersectingHandles` as done here or update the function itself to make this parameter optional?